.changeset/heavy-frogs-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-node': patch
+---
+
+Fix the scheduler "max listeners exceeded" warning by having "abort controllers" per task instead of having a single one for all tasks (previously we used the same controller for all collators, logging the warning every time we had more than 10 collators running in parallel).
```

Comment:
Maybe an alternative message that puts the focus on _what_ was fixed rather than _how_ it was fixed could be...

> Fixed a bug that could cause a `max listeners exceeded` warning to be logged when more than 10 collators were running simultaneously.

Reply:
Updated! Thank you for suggesting 🙌🏻